humble/distribution.yaml

```diff
@@ -6529,8 +6529,8 @@ repositories:
     release:
       tags:
         release: release/humble/{package}/{version}
-      url: https://github.com/ros2-gbp/septentrio_gnss_driver_ros2-release.git
-      version: 1.2.3-1
+      url: https://github.com/septentrio-users/septentrio_gnss_driver_ros2-release
```

Comment:
See my comments in https://github.com/ros/rosdistro/pull/37683#pullrequestreview-1488743367